modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/CQLBasedAndesContextStoreImpl.java

```diff
@@ -532,4 +532,41 @@ private ResultSet execute(Statement statement, String task) throws AndesExceptio
             throw new AndesException("Error occurred while " + task, e);
         }
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void updateExclusiveConsumerForQueue(String queueName, boolean isExclusiveConsumer) throws AndesException {
+
+        //retrieving queue information
```

Comment:
Thanks. I'll do the changes

On Thu, Apr 2, 2015 at 9:52 AM, Sajini De Silva <notifications@github.com>
wrote:

> In
> modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/CQLBasedAndesContextStoreImpl.java
> <https://github.com/wso2/andes/pull/93#discussion_r27631826>:
>
> > @@ -532,4 +532,41 @@ private ResultSet execute(Statement statement, String task) throws AndesExceptio
> >              throw new AndesException("Error occurred while " + task, e);
> >          }
> >      }
> > +
> > +    /**
> > +     * {@inheritDoc}
> > +     */
> > +    @Override
> > +    public void updateExclusiveConsumerForQueue(String queueName, boolean isExclusiveConsumer) throws AndesException {
> > +
> > +        //retrieving queue information
>
> Start the comments with capital letters
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wso2/andes/pull/93/files#r27631826>.
>



-- 

Chamalee De Silva

Software Engineer | WSO2

E:chamalee@wso2.com <niroshika@wso2.com>
M :0711540738| http://wso2.com <http://wso2.com/>